eblob_kit.py

```diff
@@ -675,19 +779,27 @@ def check_hole(self, position, end):
             self.holes += 1
             self.holes_size += end - position
 
+
     def resolve_mispositioned_record(self, header_idx, position, valid_headers):
-        """
-        Try to resolve mispositioned record failure.
+        """Try to resolve mispositioned record failure.
 
         Return whether header at @header_idx should be skipped.
         """
         header = self.index_headers[header_idx]
 
-        assert header_idx > 0, 'Mispositioned record failure can not occur with first header'
+        # TODO(karapuz): errors instead of asserts?
+        assert \
```

Comment:
What and why was changed here?

Reply:
PEP: max. line width exceeded.

Reply:
Please, use 120 symbols for max. line width